### PR TITLE
Patch limited access

### PR DESCRIPTION
**Pull request by Jopyth**
Solves #456 as discussed there. 

Features:
- Warning message of default behaviour is logged, if `ipWhitelist` value is not specified in the `config`
- If you try access from a blacklisted IP you will see how to change it
- It logs if anyone tries to access your mirror, who is not allowed

From my PC:
screenshot from 2016-09-29 17 10 32

From my phone:
screenshot_2016-09-29-17-10-56

### CODE REVIEW

File: js/server.js [hunk 1]
```diff
@@ -10,11 +10,26 @@ var app = require("express")();
 var server = require("http").Server(app);
 var io = require("socket.io")(server);
 var path = require("path");
+var ipfilter = require('express-ipfilter').IpFilter;
 
 var Server = function(config, callback) {
 	console.log("Starting server op port " + config.port + " ... ");
 
 	server.listen(config.port);
+	if (config.ipWhitelist === undefined) {
```
Comment (MichMich) [inline js/server.js]: The defaults for the config can be set in: https://github.com/MichMich/MagicMirror/blob/develop/js/defaults.js that might be a cleaner option.

**Comment by MichMich** [comment]
Awesome work! I've added a comment about the default settings. You might want to change/look into that. Also, is it possible for you to add information about this feature to the readme and add it to your PR?

File: js/server.js (hunk 1 above)
Comment (Jopyth) [inline js/server.js]: Yes, that makes sense. However can we still display the warning message then?

**Comment by MichMich** [comment]
Also, it seems like travis bot isn't happy:

````
  13:24  error  Strings must use doublequote  quotes
  25:52  error  Strings must use doublequote  quotes
````

You might want to check the code using `grunt`.

File: js/server.js (hunk 1 above)
Comment (MichMich) [inline js/server.js]: Just check if it's set to a certain value, and then show a message ...

**Comment by Jopyth** [comment]
Will take care of this. Maybe we can do it without the warning message. If someone wants to access the mirror from outside, the error message is pretty clear on how to fix it.

**Comment by MichMich** [comment]
Awesome work! Thanks!

**Comment by MichMich** [comment]
I just tried installing develop on my real mirror, and get `Access denied to IP address: ::1` with the default config. Any idea?

**Comment by MichMich** [comment]
Never mind, fixed: https://github.com/MichMich/MagicMirror/commit/b267491934fcdc46e361f4876cf201a2c8785314

**Comment by Jopyth** [comment]
I had `::1` at first (since that is wikipedia said would be the loopback address), but my IPv6 somehow displayed `::ffff:127.0.0.1`, so I changed it to that instead. Lets hope this works for everyone now.

**Pull request merged**